package.json

```diff
@@ -99,6 +103,21 @@
           "group": "navigation"
         }
       ]
+    },
+    "configuration": {
+      "title": "Auto Commit Msg",
+      "properties": {
+        "autoCommitMsg.useTitlecaseDescription": {
```

Comment:
Please remove the titlecase behavior from your PR? That was on a feature branch / PR that is not ready.  https://github.com/MichaelCurrin/auto-commit-msg/pull/81

Can you make your changes on `master` rather?

Or do `git revert` on the first the 3 commits here which not relevant
![image](https://user-images.githubusercontent.com/18750745/203154497-a2d27fc7-6980-4281-978e-30c249ff0242.png)

https://github.com/MichaelCurrin/auto-commit-msg/pull/86/commits